### ISSUE TITLE
Update marquee, timer, aria-live=off to match the currently reality supported by all major browsers and screen readers

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#2674]
## IDL questions re: aria-live=off.

https://w3c.github.io/aria/

The spec defines aria-live as a global attribute in https://w3c.github.io/aria/#aria-live
> Used in Roles:	All elements of the base markup

it also defines `off` as the default value:
> `off` (default):	Indicates that updates to the region should not be presented to the user unless the user is currently focused on that region.

The definition "should not be presented to the user _unless_ the user is currently focused on that region" accurately reflects the intention based on my recollection at the time, as well as based on other prose in the spec about order prioritization, etc. 

> Priority levels (none[sic, errata, should be 'off'] , polite, assertive) act as an ordering mechanism for updates and serve as a recommendation to user agents or assistive technologies. 

However, the combination of this default value, its definition, and the global attribute means that all elements are live regions at all times... And there is no real way to stop live region announcements from the element the AT user is focused on. "off" doesn't really mean "off"... 

I don't believe the editors intended to make aria-live affect elements globally this way, at least not in the context of `aria-live=off`, or didn't consider the consequences of making a value other then undefined the default value for a global attribute. (I was one of the spec editors around that time and admit to not considering this consequence in the context of an IDL implementation.)

I expect this might be addressed as errata... 

~~The easiest and most web-compatible option may be to change `aria-live` to no longer be global. Instead, defining it and other live region attributes as allowed only on elements with an explicitly defined live region role, and ignored on other elements.~~

~~Another solution may be to add another `undefined` value as the default, but I think this path has more risk to webcompat.~~

[Update: I struck the previous two options because the [third option](https://github.com/w3c/aria/issues/2674#issuecomment-3555965835) below seems best now]

**Comment by cookiecrook** [comment]
@rahimabdi FYI ~~re: IDL since in either case, there should be at least an "undefined" or "no state" state in the IDL... Implementations will still need to parse this "undefined" state in order to support the "default value for role."~~

Nevermind. See below.

**Comment by cookiecrook** [comment]
More here:
- https://github.com/w3c/aria/issues/2144

**Comment by cookiecrook** [comment]
We just verified [these 3 test cases (timer, marquee, div[aria-live=off])](https://bug-271841-attachments.webkit.org/attachment.cgi?id=470641) trigger no speech in VoiceOver, JAWS, NVDA, and Narrator on Safari, Chrome, Firefox, and Edge. I recall the results were different a few years ago, but regardless, we have alignment now.

It therefore seems like the best option is to consider changing the spec to align with the all the implementations. Especially given the advent of ariaNotify, and our current inability to write WPT tests for live regions, it seems unwise to move the engines out of alignment.

I don't think this will require a change to IDL, since effectively `off` and `undefined` are treated the same. 

This will, however, result in the following substantive changes to the spec though.
- `marquee` and `timer` should no longer be listed as live regions, as the default `off` value effectively always made them static elements, and the implementations reflect that.
- the definition of the `aria-live` value for `off` should be updated to remove the implication that notifications will work when the AT is focused on the region: 

> off (default): Indicates that updates to the region should not be presented to the user ~~unless the user is currently focused on that region~~.

**Comment by cookiecrook** [comment]
Agenda+ just to make sure the PR doesn't slip by any WG members accidentally.

**Comment by daniel-montalvo** [comment]
@cookiecrook Nothing to add from me here. I tried this with regular divs without roles and braille still updates, so as @mcking65 was saying this is expected behavior.

**Comment by cookiecrook** [comment]
Yes, thanks... As long as the behavior is the same between [the 3 test cases (timer, marquee, div[aria-live=off])](https://bug-271841-attachments.webkit.org/attachment.cgi?id=470641), then that is expected behavior presumably unrelated to aria-live... I'll try to verify the additional braille results with another test case on Mac and Windows before merging.

**Comment by cookiecrook** [comment]
No unexpected results in Braille either. @spectranaut will test Orca speech output before merging the PR.
